src/test/java/com/zombiecastlerush/building/ItemFactoryTest.java

```diff
@@ -0,0 +1,118 @@
+package com.zombiecastlerush.building;
+
+import com.fasterxml.jackson.core.JsonProcessingException;
+import com.fasterxml.jackson.databind.ObjectMapper;
+import org.junit.Assert;
+import org.junit.Before;
+import org.junit.Test;
+
+import java.io.File;
+import java.io.IOException;
+import java.util.ArrayList;
+import java.util.List;
+
+public class ItemFactoryTest {
+    private List<Item> items = new ArrayList<>();
+    private String outputPath = System.getProperty("user.dir").concat("\\target\\items.json");
+    ;
+
+    @Before
+    public void setUp() throws Exception {
+        ObjectMapper mapper = new ObjectMapper();
+        System.out.println(outputPath);
+        File file = new File(outputPath);
+        for(int i = 0; i < 10; i ++){
+            Item item = new Item("item"+i, "i am #"+i + " item");
+            items.add(item);
+        }
+        ItemFactory.writeItemsToDir(file, items); // generate .json file
+    }
+
+    @Test
+    public void testReadItemsFromDirectory_returnAListOfItems() throws IOException {
+        Inventory inventory = new Inventory();
+        File file = new File(outputPath);
```

Comment:
Does this need to be done in both setUp() and in every method?

Reply:
Will test and fix